starters/blog/gatsby-node.js

```diff
@@ -12,15 +12,21 @@ exports.createPages = async ({ graphql, actions, reporter }) => {
     `
       {
         allMarkdownRemark(
-          sort: { fields: [frontmatter___date], order: DESC }
+          sort: { fields: [frontmatter___date], order: ASC }
```

Comment:
Why changing the order here?

On the index page we have `DESC`: https://github.com/gatsbyjs/gatsby/blob/master/starters/blog/src/pages/index.js#L75

Reply:
This is because current code already invert previous to next (and vice versa) in https://github.com/gatsbyjs/gatsby/blob/d20d2d20182d8a31a0a39cdf3beec61dc0500987/starters/blog/gatsby-node.js#L47-L48

Because I drop that code completely, I need graphql `previous` and `next` to already be correct.

Order in this query doesn't have impact other than that

Reply:
(without `order` change here the result would be that next/prev links at bottom of details page - for example in https://gatsby-starter-blog-demo.netlify.app/new-beginnings/ would be inverted)

Reply:
I guess I could invert them when passing context but:
```
          next: post.previous ? post.previous.id : undefined,
          previous: post.next ? post.next.id : undefined,
```
would be more confusing?

Reply:
Ok, wanted to double check that as we once had that exact issue of inverted links 👍